config/nginx/docker.conf

```diff
@@ -0,0 +1,21 @@
+server {
```

Comment:
I understand now :) I reviewed you pull request super late last night.

I'm going to merge this and work on a few modifications (Ubuntu 16.04 for Vagrant, updating composer dependencies).